.circleci/config.yml

```diff
@@ -16,6 +16,17 @@ jobs:
           name: "test"
           command: |
               ./test.sh
+      - run:
+          name: Build Docker images
+          command: |
+              ./build-images.sh 11
+              ./build-images.sh 14
+      - run:
+         name: Login to Docker
```

Comment:
What are we trying to achieve by adding this step ?

Reply:
We are building docker images with node versions baked inside it. These images will be pushed into dockerhub fnproject profile for clients to pull the latest node version docker images.